rtv/content.py

```diff
@@ -143,6 +143,20 @@ def strip_praw_submission(sub):
 
         return data
 
+    @staticmethod
+    def strip_praw_subscription(subscription):
+        """
+        Parse through a subscription and return a dict with data ready to be
+        displayed through the terminal.
+        """
+
+        data = {}
+        data['object'] = subscription
+        data['type'] = 'Subscription'
+        data['name'] = "/r/" + subscription._case_name
```

Comment:
Any reason we can't use ``subscription.display_name``? I would rather avoid private variables if possible.